pkg/pagerduty/pagerduty.go

```diff
@@ -23,8 +22,8 @@ const (
 type PagerDuty struct {
 	// c is the PagerDuty client
 	c *sdk.Client
-	// userEmail is the user that will run the commands
-	userEmail string
+	// currentUser is the current logged in user
```

Comment:
We don't need the currentUser. We will always be passing in Service Account Credentials